src/Data/Graph/Typed.hs

```diff
@@ -79,6 +81,29 @@ fromAdjacencyIntMap (AIM.AM m) = GraphKL
   where
     (g, r, t) = KL.graphFromEdges [ ((), v, IntSet.toAscList us) | (v, us) <- IntMap.toAscList m ]
 
+-- | Build 'GraphKL' from a 'LabelledAdjacencyMap'.
+-- If @fromLabelledAdjacencyMap g == h@ then the following holds:
+--
+-- @
+-- map ('fromVertexKL' h) ('Data.Graph.vertices' $ 'toGraphKL' h)                               == 'Algebra.Graph.AdjacencyMap.vertexList' g
+-- map (\\(x, y) -> ('fromVertexKL' h x, 'fromVertexKL' h y)) ('Data.Graph.edges' $ 'toGraphKL' h) == 'Algebra.Graph.fromLabelledAdjacencyMap.edgeList' g
+--
+-- 'toGraphKL' (fromLabelledAdjacencyMap (1 * 2 + 3 * 1)) == 'array' (0,2) [(0,[1]),(1,[]),(2,[0])]
+-- 'toGraphKL' (fromLabelledAdjacencyMap (1 * 2 + 2 * 1)) == 'array' (0,1) [(0,[1]),(1,[0])]
+-- @
+fromLabelledAdjacencyMap :: Ord a => LAM.LabelledAdjacencyMap a e -> GraphKL a
```

Comment:
I think we should instead add a conversion `LabelledAdjacencyMap a e -> AdjacencyMap a` to the `LabelledAdjacencyMap` module and then there is no need to extend this one.